### ISSUE TITLE
Problem in ASP.NET with multiple pickers

### ISSUE THREAD

**Issue by Cyberherbalist** [jonthornton/jquery-timepicker#374]
In ASP.NET, when I am trying to do only one textbox as per this:

     <asp:TextBox ID="TextBox1" runat="server"></asp:TextBox>
     <script>
         $(function () {
             $('#TextBox1').timepicker();
         });
     </script>

It works just fine. Everytime I click into the textbox it pops the dropdown and the selection goes into the textbox without fail.

But when I try having TWO timepickers:

     <asp:TextBox ID="TextBox1" runat="server"></asp:TextBox>
     <br />
     <asp:TextBox ID="TextBox2" runat="server"></asp:TextBox>
     <script>
         $(function () {
             $('#TextBox1').timepicker();
             $('#TextBox2').timepicker();
         });
     </script>

... and then comes odd behavior. If I click outside the two controls before I click on either textbox, neither one pops the timepicker. If I click either of the controls, but don't select a time, then after that neither one pops a timepicker. If I click on either one of the textboxes and actually select a time, then from then on they will both pop timepickers, even if I never select a time again.

Also, when running inside Visual Studio 2013 only, I first get an error dialog which says:

     Unhandled exception at line 65, column 3427 in
     http://localhost:1459/aa66b45d9cf7427b8b7658d9362cd16a/browserLink
     0x800a01b6 - JavaScript runtime error: Object doesn't support property or method 'bind'

**Comment by jonthornton** [comment]
Wow, that sounds pretty bizarre. Unfortunately I'm not very familiar with ASP. Can you provide a working demo of the generated html/js, as a jsFiddle or source files?

**Comment by markvantilburg** [comment]
Tried it and have no issues with timepicker and asp.net:

my code:
```
<asp:TextBox ID="TextBox1" clientidmode="static" runat="server"></asp:TextBox>
 <br />
 <asp:TextBox ID="TextBox2" clientidmode="static" runat="server"></asp:TextBox>

 <script>
     $(function () {
         $('#TextBox1').timepicker();
         $('#TextBox2').timepicker();
     });
 </script>
```

What is your clientIdMode setting for those textboxes? If they are not static they change after a postback and then the id's would not match.

Also the error for browerLink seems to be from 'web essentials' plugin if i'm correct.

**Comment by markvantilburg** [comment]
my sample above renders this html

```
<input name="TextBox1" type="text" id="TextBox1" />
 <br />
 <input name="TextBox2" type="text" id="TextBox2" />

 <script>
     $(function () {
         $('#TextBox1').timepicker();
         $('#TextBox2').timepicker();
     });
 </script>
```

**Comment by Cyberherbalist** [comment]
Well, after I carefully prepared a demo of the problem in order to post on one of my personal websites, I tested it only to find that I couldn't duplicate the bad behavior!  The behavior does occur while running it in Visual Studio 2013, however.  If you want to see it, I've provided the source code to the page in question, so navigate to http://clarkmichael.com/TimePicker2.aspx.  The source code is displayed as text in the page itself.

So, I don't think this can rightfully called a timepicker bug, per se.
